### PR TITLE
Javascript Objects Implementation - Part 1

### PR DESCRIPTION
**Pull request by cancelei**
Hello Reviewer, 

In this project, we implemented a simple website to store Books.

**Key Features**
- Remove button;
- Add button;
- LocalStorage to store items, when an item is removed, it is removed from the Local Storage as well;

Happy review.

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented:
- Remove the Live demo link from the ReadMe if the app is not yet deployed. Similarly, I urge you to remove the Live Video link as the video is not yet present.
- Ensure to remove the Microverse template logo. Optionally, you can replace it with your own logo if you have one.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  e.preventDefault();
+
+  const bookTitle = addBook.title.value.trim();
+  const bookAuthor = addBook.author.value.trim();
+
+  if (bookTitle.length && bookAuthor.length) {
+    bookCollection.push({ title: bookTitle, author: bookAuthor });
+    localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
+    renderBooks();
+
```
Comment (Ismailco) [inline app.js]: - [ ] Kindly consider changing this into a function that add a new book to the collection, then you can use the new function inside the event listner

File: app.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+});
+
+bookShelf.addEventListener('click', (e) => {
+  if (e.target.classList.contains('remove')) {
+    const { index } = e.target.dataset;
+    bookCollection.splice(index, 1);
+    localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
+    renderBooks();
+  }
+});
```
Comment (Ismailco) [inline app.js]: - [ ] Same here kindly make sure to create a function for removing a book from the collection

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @cancelei, @rloterh, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @mohashyne & @cancelei 

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ `add` and `remove` functions are working, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your README looks professional, well done!
-  ✅ Data is preserved, good work.
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**